packages/cfd/src/Components/cfd-real-account-display.tsx

```diff
@@ -200,7 +200,7 @@ const CFDRealAccountDisplay = ({
             platform={platform}
             descriptor={
                 platform === 'mt5'
-                    ? localize('Trade CFDs on our synthetics and basket indices.')
+                    ? localize('Trade CFDs on our synthetics, baskets, and derived FX.')
```

Comment:
Where is this being used on the page?

Reply:
On mt-5, platform dashboard page. Can be seen even without logging in. If you are logged in and don't have mt-5 account it shows a button "Add real account"

https://app.deriv.com/mt5#real <==== for reference